org.aposin.licensescout.core/src/test/java/org/aposin/licensescout/util/JsonUtilTest.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2019 Association for the promotion of open-source insurance software and for the establishment of open interface standards in the insurance industry (Verein zur Förderung quelloffener Versicherungssoftware und Etablierung offener Schnittstellenstandards in der Versicherungsbranche)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
```

Comment:
Including the following code suggestion will solve it:
```suggestion
```

The main issue is that there is an automatic check for the license header, and it should be formatted exactly as expected. There are several options, for future contributions, that you can use to make sure that this don't happen:

* Run on `org.aposin.licensescout.core` the `mvn license:format` goal to update any missing/outdated header. You can try it out instead of accepting the suggestion and commit the changes.
* If you are using Eclipse as an IDE, we included a launch configuration to launch that goal. You can also try this option out.
* Another option is to copy as-is the header from other files; or setup your environment to include it in every new file it appears.